backend/src/schema/types/type/User.gql

```diff
@@ -68,10 +68,11 @@ type User {
   RETURN COUNT(u) >= 1
   """
   )
+  
   isBlocked: Boolean! @cypher(
   statement: """
-  MATCH (this)<-[: BLOCKED]-(u: User { id: $cypherParams.currentUserId})
-  RETURN COUNT(u) >= 1
+  MATCH (this)<-[:BLOCKED]-(user:User {id: $cypherParams.currentUserId})
```

Comment:
```suggestion
  MATCH (this)-[:BLOCKED]-(user:User {id: $cypherParams.currentUserId})
```
I would argue `BLOCKED` relationship is vice versa and then we can refactor the code below :point_down:

Reply:
I was working under the understanding that blocking someone means only that a user can't interact with my content... Therefore not reciprocal